bots/tests-invoke

```diff
@@ -166,9 +166,9 @@ class PullTask(object):
             traceback.print_exc()
             return "Rebase failed"
 
-        # COMPAT: If the bots directory doesn't exist in this branch, check it out from master
+        # Always use bots/ and test images from master
         try:
-            if subprocess.call([ "git", "ls-tree", "-d", "HEAD:bots/"], stdout=DEVNULL) != 0:
+            if not self.base or self.base != "master":
```

Comment:
I think this should be:

```
if self.base and self.base != "master":
```

When self.base is not set, it's typically a one off or custom invocation.